### DISCUSSION TITLE
Smaller Icons at the bottom

### DISCUSSION THREAD

**Discussion by dtetreau251** [Vets-Who-Code/march2021#89] [Ideas]
Can we make smaller icons at the bottom?

**Comment by stephanlamoureux** [comment]
That's part of the footer styling I'm working on

**Reply by dtetreau251** [reply]
Cool I'll make it an issue

**Comment by dtetreau251** [comment]
this was handled by Steve

**Comment by joeromine** [comment]
I didn't make them smaller but I put them in line with the coded by
